shared/entity/type.go

```diff
@@ -158,7 +170,21 @@ var entityTypes = map[Type]typeInfo{
 	TypeIdentityProviderGroup: identityProviderGroup{},
 }
 
-type container struct{}
+// APIMetricsEntityTypes returns a slice containing the entity types that are relevant for the API metrics.
+func APIMetricsEntityTypes() []Type {
```

Comment:
This should include TypeServer too right?

Reply:
Yes, it did because I was defining some prefixes for TypeServer as well (namely `resouces`, `metrics`, `events` and `metadata`). But your question prompted me to not define prefixes for it since it will be used as a default value anyway.